### PR TITLE
#415-improving card items

### LINKED ISSUE #415: [FEATURE] Add a hover effect on post items.
### Description

We should add a hover effect on post items as static card are not that great.
We can add a ease in or ease out kind of effect.

### Screenshots

scrnli_02_06_2023_18-58-37

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Soulpurr**
I have added a hover effect which will ease out kind on effect on hovering.
fixes #415 

[scrnli_03_06_2023_16-18-39.webm](https://github.com/narayan954/dummygram/assets/92094765/21152b4d-cd57-4cd1-b626-3f825ccfaf82)

### CODE REVIEW

**Comment by Soulpurr** [comment]
@narayan954  pls verify and approve this so that i can create another pr for the other issue in order to avoid iany conflicts.

**Comment by narayan954** [comment]
> @narayan954 pls verify and approve this so that i can create another pr for the other issue in order to avoid iany conflicts.

okay sure, on it

File: src/index.css [hunk 1]
```diff
@@ -69,6 +69,14 @@ input:focus {
   border-radius: 5px;
   margin-bottom: 20px;
   word-wrap: break-word;
+  cursor: pointer;
```
Comment (narayan954) [inline src/index.css:72]: cursor pointer is not the property we'd want. post card is not a clickable or an action... it's just a card to view and items inside it are clickable actually

**Review by narayan954 — APPROVED** [review]
it needs just a tiny change, that I'm fixing myself. Other than that it looks great!

**Comment by narayan954** [comment]
Thanks for your contribution! :)

**Pull request merged**